code/game/machinery/doors/windowdoor.dm

```diff
@@ -167,18 +167,16 @@
 	..()
 
 //When an object is thrown at the window
-/obj/machinery/door/window/hitby(AM as mob|obj)
-
+/obj/machinery/door/window/hitby(atom/movable/AM)

```

Comment:
All these hitby's that would work out better for the damage system.

One day....one day....